File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

void space(int n);
void block(int n);

int main(void)
{
    
    //Get height from user
    int height;
    do 
    {
        height = get_int("How tall?\n");
        
    }
    //ensure it's between 1 and 8 inclusive
    while (height < 1 || height > 8);
    int i;
    int h = height;
    //alternate functions to create appropriate pyramid height
    for (i = 1; i < (height + 1); i++, h--)
    {
        space(h);
        block(i);
        printf("  ");
        block(i);
        printf("\n");
        
    }
    
    
}
//function to create appropriate number of space
void space(int n)
{ 
    for (int i = 0; i < (n - 1); i++)
    {
        printf(" ");
        
    }
   
}
//Function to create hashes
void block(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
      
    }
    
}

File: pset6/mario/more/mario.py
from cs50 import get_int


# declare height variable
height = 0

# get hieght from user enforcing it be between 1 and 8
while height < 1 or height > 8:
    height = get_int("how tall?\n")

# create dummy variable for height to manipulate
h = height

# loop through height number of times to create pyramid
for i in range(1, height+1):
    # print spaces
    print(" " * (h - 1), end="")

    # reduce h to reduce spaces to print
    h -= 1

    # print first block
    print("#" * i, end="")

    # print space divider
    print("  ", end="")

    # print second set of blocks
    print("#" * i)

# can be done neater with recurssion
